pkg/tests/conftest.py

```diff
@@ -393,10 +459,24 @@ def test_account(salt_call_cli):
 
 
 @pytest.fixture(scope="module")
-def salt_api(salt_master, install_salt):
+def extras_pypath():
+    extras_dir = "extras-{}.{}".format(*sys.version_info)
+    if platform.is_windows():
+        return pathlib.Path(
+            os.getenv("ProgramFiles"), "Salt Project", "Salt", extras_dir, "bin"
+        )
+    elif platform.is_darwin():
+        return pathlib.Path(f"{os.sep}opt", "salt", extras_dir, "bin")
```

Comment:
```suggestion
        return pathlib.Path(f"/opt", "salt", extras_dir, "bin")
```
This is nitpicking, but we know that it will always be `/`